CIME/case/case_clone.py

```diff
@@ -219,8 +219,6 @@ def create_clone(
             )
         )
 
-        newcase.case_setup()
```

Comment:
Did this have anything to do with the SMP_PRESENT to BUILD_THREADED change?

Reply:
I think it was something I caught as being out of place during the development of that change, so it's not directly related to that but it was probably included in that PR.   If you want to put it back and then open an issue to fix the problem by fixing the tests then I guess we can do it that way...

Reply:
Yes I'd like to do it that way.

Reply:
I think that we have a solution now that fixes the tests and leaves this out - are you okay with that?

Reply:
Yes.